crates/ln-dlc-node/src/config.rs

```diff
@@ -113,7 +117,7 @@ pub fn coordinator_config() -> UserConfig {
         accept_forwards_to_priv_channels: true,
         // The coordinator automatically accepts any inbound channels if they adhere to its channel
         // preferences.
-        manually_accept_inbound_channels: false,
+        manually_accept_inbound_channels: true,
```

Comment:
❓ Why did this change? Let's update the comment accordingly.

Reply:
This flag has to be set to true, if we want to be able to accept anchored channels. 

> Note that if you wish accept inbound channels with anchor outputs, you must enable UserConfig::manually_accept_inbound_channels and manually accept them with ChannelManager::accept_inbound_channel. This is done to give you the chance to check whether your reserve of onchain funds is enough to cover the fees for all existing and new channels featuring anchor outputs in the event of a force close.

I will update the docs accordingly.